content-test/selectors/siteMetadataSelectors.test.js

```diff
@@ -82,10 +82,15 @@ describe("siteMetadataSelectors", () => {
 
       assert.ok(result.parsedUrl);
     });
-    it("should set label prop", () => {
+    it("should set label prop without eTLD set", () => {
       let result = selectSiteProperties(validSpotlightSite);
 
-      assert.ok(result.label);
+      assert.equal(result.label, "cnn.com");
```

Comment:
An adjacent test `colorSelectors.test.js` checks things like `assert.equal(state.label, "foo");` and `assert.equal(state.backgroundColor, "rgba(11, 11, 11, 1)");`.

Changing test data requiring changes to tests doesn't seem that unexpected.